addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -246,7 +269,9 @@ options.registry.blog_cover = options.Class.extend({
                 return self.$filter.hasClass($(this).data('filterColor'));
             }).addClass('active').data('filterColor');
 
-        this.$target.attr('data-coverClass', this.$el.find('.active[data-select-class]').data('selectClass') || '');
+        this.$target.attr('data-coverClass', this.$el.find('.active[data-coverOpt="size"]').data('selectClass') || '');
+        this.$target.attr('data-textSizeClass', this.$el.find('.active[data-coverOpt="text_size"]').data('selectClass') || '');
+        this.$target.attr('data-textAlignClass', this.$el.find('.active[data-coverOpt="text_align"]').data('selectClass') || '');
```

Comment:
- Camel case in JS, no camel case in XML
- The role of the `_setActive` class is not to write on the target, this should probably be the role of an option method... but I am not sure of what you are doing here, why is this needed ?

Reply:
Here we save (or reset) options directly to the element as DOM attributes. These info will be read by the `_saveElement` function and stored in the DB.

I wasn't sure about this whole process, but it's what's currently used to store the   `data-select-class`, so I kept it.

Is what you're asking for?

Reply:
I don't mind the system, just the location where it is done, it should not be done in `_setActive`... but I see it is already like that since forever. So let's keep it for now. However, data-coverClass & cie should still be converted to data-cover-class, etc.